python/samples/concepts/auto_function_calling/chat_completion_with_auto_function_calling.py

```diff
@@ -9,6 +9,7 @@
 from semantic_kernel.core_plugins.math_plugin import MathPlugin
 from semantic_kernel.core_plugins.time_plugin import TimePlugin
 from semantic_kernel.functions import KernelArguments
+from semantic_kernel.prompt_template.prompt_template_config import PromptTemplateConfig
```

Comment:
```suggestion
from semantic_kernel.prompt_template import PromptTemplateConfig
```
In samples, we want to show the min-allowed import path. Would be the same for other sample imports, as well.